helper/pdfGeneration.js

```diff
@@ -344,8 +350,37 @@ exports.instanceObservationPdfGeneration = async function instanceObservationPdf
                             radioArray.push(element);
                         }
                     }))
-                }
 
+                    await Promise.all(ele.answers.map(element => {
```

Comment:
@borkarsaish65 Please include comments for better readability

Reply:
done